main.go

```diff
@@ -124,6 +132,22 @@ func generate(in configInput) ([]byte, error) {
 			},
 		},
 	}
+	if in.enableFileWatcherConfig {
+		c.CertificateProviders = map[string]certificateProviderConfig{
+			"google_cloud_private_spiffe": {
+				PluginName: "file_watcher",
+				Config: privateSPIFFEConfig{
+					CertificateFile:   "/var/run/gke-spiffe/certs/certificates.pem",
+					PrivateKeyFile:    "/var/run/gke-spiffe/certs/private_key.pem",
+					CACertificateFile: "/var/run/gke-spiffe/certs/ca_certificates.pem",
+					// The file_watcher plugin will parse this a Duration proto, but it is totally
+					// fine to just emit a string here.
+					RefreshInterval: "10m",
```

Comment:
Can we generate "600s" instead of "10m"? Just realized the Java [`Durations.parse(String)`](https://github.com/protocolbuffers/protobuf/blob/master/java/util/src/main/java/com/google/protobuf/util/Durations.java#L241) only supports "s" as the suffix :-(

Reply:
That should be true for all languages, as that's the protobuf Duration JSON format https://github.com/protocolbuffers/protobuf/blob/0bb6dd8fd77282cdc04c26fa01dbddaec71ac580/src/google/protobuf/duration.proto#L92 .

Reply:
If that's true we should not bother with this Duration stuff at all. Just make it an int and call it `refresh_interval_secs` to make it easy. We thought it was possible to use various suffixes like "m", "d" for minutes or days etc but that's not there.

Reply:
No, we've consistently used Duration for durations in our JSON. Doing something else just adds confusion.

Reply:
Fixed it. Sorry for the confusion. I don't know why I assumed it would support `m`, `d`, `h` etc.